cdk/.eslintignore

```diff
@@ -2,3 +2,5 @@ node_modules
 cdk.out
 .eslintrc.js
 jest.config.js
+**.d.ts
```

Comment:
Check if this is the best way to respond to this error:

 0:0  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/bin/cdk.d.ts` using `parserOptions.project`: /users/akinsola_lawanson/code/consent-management-platform/cdk/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file

Reply:
I think that you would want to edit CDK’s `tsconfig.json` instead.

Reply:
Do you how I would do this? I tried adding it to the exclude but that didn't work.

Reply:
Thanks this has been resolved.